openeo/rest/datacube.py

```diff
@@ -1141,6 +1141,40 @@ def reduce_dimension(
             metadata=self.metadata.reduce_dimension(dimension_name=dimension) if self.metadata else None,
         )
 
+    @openeo_process
+    def reduce_spatial(
+        self,
+        reducer: Union[str, typing.Callable, UDF, PGNode],
+        context: Optional[dict] = None,
+        process_id="reduce_spatial",
+        band_math_mode: bool = False,
```

Comment:
Is there a reason to expose `process_id` and `band_math_mode` here? Or is that from copying `reduce_dimension`. If the latter: don't provide these arguments